willywax/NextProduct
# #168021502 Use database connection string set as environmental variable.

#### What does this PR do?
Fixes use of environmental variables for database connection
#### Description of Task to be completed?
- Require babel register in .sequelizerc file to use ES6 in the config file
- Change the confing.json to config.js
- Require dotenv package at the top of the config file
#### How should this be manually tested?
- Clone this repo and checkout this branch
- Start your database server
- Run `sequelize db:migrate:undo:all` and see if tables are dropped
- Run `sequelize db:migrate` to create the tables
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#168021502](https://www.pivotaltracker.com/story/show/168021502)
